platforms/nuttx/NuttX/CMakeLists.txt

```diff
@@ -295,7 +295,7 @@ function(add_nuttx_dir nuttx_lib nuttx_lib_dir kernel extra)
 
 	add_custom_command(OUTPUT ${NUTTX_DIR}/${nuttx_lib_dir}/lib${nuttx_lib}.a
 		COMMAND find ${nuttx_lib_dir} -type f -name *.o -delete
-		COMMAND make -C ${nuttx_lib_dir} ${nuttx_build_options} --no-print-directory all TOPDIR=${NUTTX_DIR} KERNEL=${kernel} EXTRAFLAGS=${extra}
+		COMMAND make -C ${nuttx_lib_dir} ${nuttx_build_options} all TOPDIR=${NUTTX_DIR} KERNEL=${kernel} EXTRAFLAGS=${extra} >> nuttx_build.log
```

Comment:
```suggestion
		COMMAND make -C ${nuttx_lib_dir} ${nuttx_build_options} all TOPDIR=${NUTTX_DIR} KERNEL=${kernel} EXTRAFLAGS=${extra} > nuttx_build.log
```

Reply:
No that results in the log missing what is needed.

Reply:
Oh, this custom command is per library. It doesn't really matter, but I would split it so we get a clean log for each library build that isn't appended so that it's easy to see what actually happened in a new build. Each NuttX library build is single threaded, but the px4 build system usually kicks off multiple in parallel.

I can do it in a followup later.